src/components/views/rooms/EventTile.tsx

```diff
@@ -1121,9 +1121,13 @@ export class UnwrappedEventTile extends React.Component<IProps, IState> {
             || Boolean(this.state.contextMenu));
 
         // Thread panel shows the timestamp of the last reply in that thread
-        const ts = this.context.timelineRenderingType !== TimelineRenderingType.ThreadsList
+        let ts = this.context.timelineRenderingType !== TimelineRenderingType.ThreadsList
             ? this.props.mxEvent.getTs()
-            : this.state.thread?.replyToEvent.getTs();
+            : this.state.thread?.replyToEvent?.getTs();
+        if (typeof ts !== "number") {
+            // Fall back to something we can use
+            ts = this.props.mxEvent.getTs();
+        }
```

Comment:
IMO this would have been more readable as

```
        let ts = this.props.mxEvent.getTs();
        if (this.context.timelineRenderingType !== TimelineRenderingType.ThreadsList &&
            this.state.thread?.replyToEvent
        ) {
            ts = this.state.thread?.replyToEvent.getTs();
        }
```

Reply:
True, this looks quite a bit better